tests/tests_create_lvmraid_then_remove.yml

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
For the sake of test speed I should probably make this test a part of one that already exists.

Reply:
Fixed. Test file removed, contents appended to `tests_create_raid_pool_then_remove.yml`.